packages/server/migrations/20221206115201_add_variants_table.js

```diff
@@ -20,7 +20,7 @@ exports.up = function (knex) {
       'orange',
     ]);
     table.enum('size', ['XS', 'Small', 'Medium', 'Large', 'XL', 'XXL', '3XL']);
-    table.binary('picture');
+    table.string('picture');
```

Comment:
Might wanna make this `pictureUrl`

Reply:
@miltonhultgren can you explain a bit more? i did not see any datatype called pictureUrl.

Reply:
I meant, `table.string('pictureUrl);` since what you will store there is the URL